ci.cjs

```diff
@@ -0,0 +1,160 @@
+#! /usr/bin/env node
+
+const fs = require("fs").promises;
+const path = require("path");
+
+function printUsage() {
+  console.log(
+    `Usage: "ci check-consistency" or "ci bump-version %postfix%"`,
+  );
+}
+
+let postfix;
+const mode = process.argv[2];
+
+function validateArgs() {
+  switch (mode) {
+    case "get-version":
+      return true;
+
+    case "bump-version":
+      postfix = process.argv[3];
+      if (!postfix) {
+        printUsage();
+        process.exit();
```

Comment:
exit(1)?

Reply:
Why? `--help` returns 0 usually

Reply:
but it's not `--help` situation, it's "invalid arguments" situation, an error